images/README.md

```diff
@@ -0,0 +1,7 @@
+#### favicon.png 
+
+Small version (200px x 200px) of official Cloudify logo. 
```

Comment:
Can add automatic verification of this?

Reply:
We can write a simple test that goes over all the image and validate their size, what do u think?

Reply:
Do we have volounteer for doing that? ;-) I think, yes, that's nice improvement, but I'd like to separate that from the documenting part.

Reply:
I think I will do it, I want to practice writing tests :)